active_discussion/unions.md

```diff
@@ -1,3 +1,29 @@
 # Unions
 
-TBD
+## Outstanding questions
+
+* Is `#[repr(Rust)]` the bag-o-bytes union repr, or do we want to propose a new repr?
+    * *Discussion:* [#73: Validity of unions][#73]
+* The following questions are all implicitly answered if `#[repr(Rust)]` is the bag-o-bytes repr, but remain open if not:
+    * Do `#[repr(Rust)]` enums guarantee all fields at offset 0?
```

Comment:
```suggestion
    * Do `#[repr(Rust)]` unions guarantee all fields at offset 0?
```
I assume all the enums here are actually unions? ;)

Reply:
Yes... good catch